app/assets/stylesheets/custom.scss

```diff
@@ -625,3 +626,129 @@ input, button, select, textarea {
   width: 70px;
   margin-top: -2px;
 }
+
+.content-item {
+  padding:30px 0;
+background-color: $white;
+}
+
+.content-item.grey {
+background-color: $grey-white;
+padding:50px 0;
+height:100%;
+}
+
+.content-item h2 {
+font-weight:700;
```

Comment:
thêm hết 1 space vào sau dấu : nhé, soát lại toàn bộ đi em
xong thì indent 2 space đầy đủ vào

Reply:
dạ vâng ạ